static/bidder-params/ix.json

```diff
@@ -27,6 +27,11 @@
       "minItems": 2,
       "maxItems": 2,
       "description": "An array of two integer containing the dimension"
+    },
+    "sid": {
```

Comment:
Please update the [dev-docs](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/ix-server.md) to reflect this new parameter.